frontend/src/toolbar/elements/Heatmap.tsx

```diff
@@ -0,0 +1,46 @@
+import h337, { Heatmap as HeatmapJS } from 'heatmap.js'
+import { useValues } from 'kea'
+import { useCallback, useEffect, useRef } from 'react'
+
+import { heatmapLogic } from '~/toolbar/elements/heatmapLogic'
+
+export function Heatmap(): JSX.Element | null {
+    const { heatmapJsData, heatmapEnabled, heatmapFilters } = useValues(heatmapLogic)
+    const h337Ref = useRef<HeatmapJS<'value', 'x', 'y'>>()
```

Comment:
h337 🤣

Reply:
but I can't have doohickies, so unfair :p

Reply:
That's what they call it in their own docs! 🙈